stories/PieChart.tsx

```diff
@@ -2,15 +2,15 @@ import * as React from 'react';
 import { storiesOf } from '@storybook/react';
 import { screenshotWrap } from '@gooddata/test-storybook';
 
-import { PieChart } from '../src';
+import { DonutChart } from '../src';
```

Comment:
I think PieChart is correct here and everywhere in this file (file is named PieChart.tsx)